projects/sonar/src/app/record/document/file/file.component.html

```diff
@@ -0,0 +1,41 @@
+<!--
+ SONAR User Interface
+ Copyright (C) 2020 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="text-center">
+  <ng-container *ngIf="file; else noFile">
+    <img src="{{ file.thumbnail }}" class="img-fluid" alt="{{ file.label }}">
+    <div class="my-1 small" *ngIf="showLabel">{{ file.label }}</div>
+    <div class="my-1">
+      <span class="badge badge-secondary text-light" *ngIf="file.restriction && file.restriction.date">
+        {{ 'Public access from' | translate }}<br>{{ file.restriction.date }}
+      </span>
+    </div>
+    <div class="my-1">
+      <a href="{{ file.links.external }}" target="_blank" *ngIf="showExternalLink && file.links.external">
+        <i class="fa fa-external-link mx-1"></i>
+      </a>
+      <a href="{{ file.links.preview }}" *ngIf="showPreview && file.links.preview" (click)="preview(file); $event.preventDefault()">
+        <i class="fa fa-eye mx-1"></i>
+      </a>
+      <a href="{{ file.links.download }}" target="_blank" *ngIf="showDownload && file.links.download">
+        <i class="fa fa-download mx-1"></i>
+      </a>
+    </div>
+  </ng-container>
+  <ng-template #noFile>
+    <img src="/static/images/no-image.png" class="img-fluid">
```

Comment:
the file name can be more explicit

Reply:
what do you think about `default-thumbnail.png`?